app/worker/process.go

```diff
@@ -142,25 +144,23 @@ func processTask(task model.Task, job *model.Job,
 				"Error: Task definition incompatible " +
 					"with specified type (http)")
 		}
-		l.Infof("processTask: calling http task: %s %s", httpTask.Method,
-			processJobString(httpTask.URI, workflow, job))
```

Comment:
why did you remove this log and moved it in HTTPTask?

Reply:
it was super expensive to process the job string twice, here for only logging and then again right after.